SDK and Build tools are updated

tasomaniac opened the pull request:
I couldn't wait :)
Support library will also be needed to be updated soon. (I think)

tasomaniac commented:
With the new build-tool it is the lint failing with 345 issues found

stefanhoth commented:
Ha, you beat me to it (but only because I added this first on a work project :wink: )

stefanhoth commented:
I'll have a look at the issues

stefanhoth commented:
Lol, what a fail. :-1: 

![image](https://cloud.githubusercontent.com/assets/45467/6587337/4340869c-c783-11e4-8d55-7c18a022615d.png)

How about releasing this before adding this lint check as an error?

stefanhoth commented:
Btw, I only see
```
Check performed at Wed Mar 11 00:10:06 CET 2015.
2 errors and 274 warnings found:
```

stefanhoth commented:
Sooo, with my lint adjustment from ccc6d95 the build passes but according to the message above it's not recommended to work like that. What do you think, @friedger @HenriqueRocha @tasomaniac?

Shall we just go for it anyway and play with it?

tasomaniac commented:
I think if we wait for the new support library to come out it will be fine.

On Wed, Mar 11, 2015, 1:38 AM Stefan Hoth <notifications@github.com> wrote:

> Sooo, with my lint adjustment from ccc6d95
> <https://github.com/gdg-x/frisbee/commit/ccc6d952a2a781bcaa5fad9c3e3d7fcc514cb73e>
> the build passes but according to the message above it's not recommended to
> work like that. What do you think, @friedger <https://github.com/friedger>
> @HenriqueRocha <https://github.com/HenriqueRocha> @tasomaniac
> <https://github.com/tasomaniac>?
>
> Shall we just go for it anyway and play with it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/302#issuecomment-78172377>.
>

tasomaniac commented:
I've just saw the Support version 22 changelog. http://developer.android.com/tools/support-library/index.html
It is about to be released. And guess what, we have AppCompat Buttons now. Yey!

stefanhoth commented:
:+1:

stefanhoth commented:
@tasomaniac The support libs dropped so I updated the PR. Hope the build will be fine now. :smile:

tasomaniac commented:
Finally! :)

tasomaniac commented:
Rebased for merge conflicts. 
Can you @all review this again? I've upgraded the other libraries as well. All of them are minor updates without an API change. 
And removed unused 2 google libraries. The application seems to be fine. Please build yourself and check before merge.

stefanhoth commented on app/build.gradle:
```diff
@@ -145,15 +145,13 @@ dependencies {
     compile 'com.google:volley:4.3'
-    compile 'com.google.http-client:google-http-client-gson:1.18.0-rc'
+    compile 'com.google.http-client:google-http-client-gson:1.19.0'
     compile 'com.google.apis:google-api-services-plus:v1-rev126-1.18.0-rc'
```
@tasomaniac do you know what this is for?

tasomaniac replied:
Which one? Http-client? 
I want to get rid of it and use joyful Retrofit everywhere. Right now, Lots of API calls depends on `google-http-client-gson` unfortunately.

stefanhoth replied:
That one `com.google.apis:google-api-services-plus:v1-rev126-1.18.0-rc`

tasomaniac replied:
I also wanted to delete it but it is used. When I upgrade it to version `1.19.0` it exceeds the 65K dex method count.

stefanhoth replied:
I was just curious what's in there what is not part of GMS.

stefanhoth commented:
Great job. :+1: 

:shipit:
